homework/questions.txt

```diff
@@ -3,72 +3,83 @@ we went through in class.
 
 1. Navigating the terminal
   a, What is a root directory?
-
+     Root directory is probably the first and top most directory in the hierarchy. The root directory is denoted by (/)slash sign
   b, What is the difference between root and home directory?
-
+     Root directory is reffered to / sign and the topmost directory and the home directory comes under the root directory.
   c, What is the difference between absolute and relative path?
-
+   Absolute path is specifting the location of the file form the root directory and it's the complete path from the start of the root directory. And Relative path is       actually reffered as the presently working directory.
```

Comment:
yes, relative path  describes the location of the file/directory relative ie. from the current working directory 🧑‍🎤🦾